### DISCUSSION TITLE
GitHub Gist does not show diff in "Preview Changes" tab

### DISCUSSION THREAD

**Discussion by sryze** [community/community#79050] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After I update a markdown gist and click on Preview Changes, I see the full file with new content instead of a diff. There is no way to see a diff to check what was added or deleted before saving the gist.

**Comment by KavindaBandara321** [comment]
GitHub Gists typically do not provide a built-in visual diff tool in the preview changes when updating a Markdown gist. The "Preview Changes" functionality in gists is more focused on allowing you to see how the rendered Markdown will look after your updates.
